pokedex/data/csv/pokemon_species_flavor_text.csv

```diff
@@ -123106,6 +123133,12 @@ a deity in Sinnoh-region mythology."
 484,26,11,"空間を　ゆがめる　能力を　持ち
 シンオウ地方の　神話では
 神様として　描かれている。"
+484,39,9,"This Pokémon is feared as a deity in Hisuian legend. The birth
+of Palkia was what caused the walls of our world to disappear,
+creating a sky that spans for infinity."
+484,39,9,"It soars across the sky in a form that greatly resembles the
+creator of all things. Perhaps this imitation of appearance is
+Palkia's strategy for gaining Arceus's powers."
```

Comment:
I see two flavor texts for the same entry. I'm assuming this pokemon has an additional entry for a form or something. I remember there being an issue about this before and the way it was resolved was for now was removing the form's entry. Same thing for id 483 and 905.
[Found the old issue](https://github.com/PokeAPI/pokeapi/issues/533)

Reply:
Thanks for addressing that. I have removed just now.